### DISCUSSION TITLE
Support for Korean Characters in Annotator Function Fonts

### DISCUSSION THREAD

**Discussion by YoungjaeDev** [roboflow/supervision#1631] [Q&A] [answered]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

I'm using the Annotator function in Roboflow Supervision and would like to add annotations in Korean. However, it seems that the default font doesn't support Korean characters. Are there any workarounds or recommended methods to enable Korean text in annotations?

Thanks in advance for your help!

**Comment by onuralpszr** [comment, marked as answer]
@YoungjaeDev :wave: Hello, Instead of workaround we already have a solution for Label Annotations and that is `RichLabelAnnotator`

[RichLabelAnnotator doc](https://supervision.roboflow.com/0.22.0/detection/annotators/#supervision.annotators.core.RichLabelAnnotator)


```python
import cv2
import supervision as sv
import

image = cv2.imread(<SOURCE_IMAGE_PATH>)

model = get_model(model_id="yolov8n-640")
results = model.infer(image)[0]
detections = sv.Detections.from_inference(results)

rich_label_annotator = sv.RichLabelAnnotator(font_path=<TTF_FONT_PATH>)
annotated_image = rich_label_annotator.annotate(scene=image.copy(), detections=detections)
```

But If you mean to run on other text require annotator like "PolygonZoneAnnotator," I am sorry we don't have support but for workaround you can do overrides to make use but for generally speaking making use case for label cases you can use RichLabelAnnotator to make it use as well. Other workaround you can put a "custom" text by making use "PIL" (Pillow) with custom font input and write what you want as well. I hope those help you out 

Thank you.

**Comment by YoungjaeDev** [comment]
@onuralpszr 

I put the TTF in and found it working fine. thanks

**Reply by onuralpszr** [reply]
Please select answer thank you
